components/contribution-flow/index.js

```diff
@@ -720,16 +757,17 @@ class ContributionFlow extends React.Component {
                     taxes={this.getApplicableTaxes(collective, host, tier?.type)}
                     onSignInClick={() => this.setState({ showSignIn: true })}
                     isEmbed={isEmbed}
-                    isSubmitting={isValidating || isSubmitted || isSubmitting}
+                    isSubmitting={isValidating || isLoading}
+                    order={this.state.submittedOrder}
                   />
                   <Box mt={40}>
                     <ContributionFlowButtons
                       goNext={goNext}
-                      goBack={goBack}
+                      goBack={isCrypto && currentStep.name === 'checkout' ? null : goBack}
```

Comment:
Why is that?

Reply:
This was added so that when the user navigates to the QR code screen, they should not be able to go back.

![image](https://user-images.githubusercontent.com/12435965/128907382-84defd3d-fba0-494b-a571-c5687974fe9f.png)

Notice there's no back button. This is sidestepping from the design a bit but we (me and Leo) had this discussion and thought was necessary as if not the user can go back and forth and each time a new order will be created. Once they come to this stage, the order is already created. It's actually similar to the final success screen in a regular donation, they can't go back, because the order is already done. Let me know if you think otherwise. 🙂

Reply:
Why don't you explain this in your code?

Reply:
Just added a comment explaining this behavior. 👍🏽

Reply:
You mean the added comment wasn't detailed enough? Currently it says,

`for crypto flow the user should not be able to go back after the order is created at checkout step`